profiles/common/modules/features/cce_basic_config/cce_basic_config.features.filter.inc

```diff
@@ -137,6 +137,39 @@ function cce_basic_config_filter_default_formats() {
     ),
   );
 
+  // Exported format: Full HTML + Change tracking.
```

Comment:
I think we do need to have consistency. The filters that come with the profiles are exported into cce_basic_config.features.filter.inc We had 4 of them, now we have 5 so why not include the 5th one?